azure-pipelines.yml

```diff
@@ -0,0 +1,24 @@
+steps:
```

Comment:
NIT: it would be good to set a build name, unless azure pipelines (can we say AP?) has made the default name meaningfull.  Again see the powershell definitions.